src/main/java/com/example/WMS/handlers/IncomingGoodsHandler.java

```diff
@@ -0,0 +1,97 @@
+package com.example.WMS.handlers;
+
+import com.example.WMS.constants.FilePaths;
+import com.example.WMS.dtos.AddStockDescriptionDto;
+import com.example.WMS.entity.IncomingGoods;
+import com.example.WMS.exceptions.IncomingGoodsNotFound;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVWriter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+@Slf4j
+public class IncomingGoodsHandler {
+    public AddStockDescriptionDto read(String receiveOrderId) {
+        try (CSVReader reader = new CSVReader(new FileReader(FilePaths.INCOMING_GOODS))) {
```

Comment:
Here also, trying using OpenCsv library for reading/writing in csv.

Reply:
using opencsv library annotations to store beans